src/components/Note/Note.jsx

```diff
@@ -1,8 +1,30 @@
 import React from "react";
+import { useNotes } from "../../context/notes-context";
+import "./note.css";
+import { FiTrash2 } from "react-icons/fi";
+import { RiInboxArchiveLine } from "react-icons/ri";
+import { MdOutlineColorLens } from "react-icons/md";
 
 export const Note = () => {
-  return <div>
-    Note
-  </div>;
+  const { note, removeFromnote } = useNotes();
+  return (
+    <div className="note-collection">
+      {note.map((item) => {
+        return (
+          <div className="note" key={item._id}>
+            <div>{item.NoteContent.title}</div>
+            <div
+              className="note-descr"
+              dangerouslySetInnerHTML={{ __html: item.NoteContent.description }}
+            />
+            <div className="flex-row note-options-set">
+              <MdOutlineColorLens onClick={() => {}}/>
+              <RiInboxArchiveLine onClick={() => {}} />
```

Comment:
the onClicks are empty is their any reason?

Reply:
will add feat in future